### ISSUE TITLE
`TypeError: Cannot set property mark of #<Object> which has only a getter`

### ISSUE THREAD

**Issue by khteh** [TypeStrong/fork-ts-checker-webpack-plugin#838]
## Current behavior

Hit the following error on Quasar Vue.JS:
```
TypeError: Cannot set property mark of #<Object> which has only a getter
TypeError: Cannot set property mark of #<Object> which has only a getter
    at Object.connectTypeScriptPerformance (/usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/typescript-reporter/profile/TypeScriptPerformance.js:12:36)
    at createTypeScriptReporter (/usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/typescript-reporter/reporter/TypeScriptReporter.js:40:49)
    at Object.<anonymous> (/usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/reporter/reporter-rpc/ReporterRpcService.js:21:30)
    at Generator.next (<anonymous>)
    at /usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/reporter/reporter-rpc/ReporterRpcService.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/reporter/reporter-rpc/ReporterRpcService.js:4:12)
    at /usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/reporter/reporter-rpc/ReporterRpcService.js:19:88
    at Object.<anonymous> (/usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/rpc/RpcService.js:23:38)
    at Generator.next (<anonymous>)
TypeError: Cannot set property mark of #<Object> which has only a getter
TypeError: Cannot set property mark of #<Object> which has only a getter
    at Object.connectTypeScriptPerformance (/usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/typescript-reporter/profile/TypeScriptPerformance.js:12:36)
    at createTypeScriptReporter (/usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/typescript-reporter/reporter/TypeScriptReporter.js:40:49)
    at Object.<anonymous> (/usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/reporter/reporter-rpc/ReporterRpcService.js:21:30)
    at Generator.next (<anonymous>)
    at /usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/reporter/reporter-rpc/ReporterRpcService.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/reporter/reporter-rpc/ReporterRpcService.js:4:12)
    at /usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/reporter/reporter-rpc/ReporterRpcService.js:19:88
    at Object.<anonymous> (/usr/src/khteh/node_modules/fork-ts-checker-webpack-plugin/lib/rpc/RpcService.js:23:38)
    at Generator.next (<anonymous>)
```
## Expected behavior

No error
## Steps to reproduce the issue

https://github.com/khteh/quasar
## Environment
 - **fork-ts-checker-webpack-plugin**: [version from the `package.json`] Not installed explicitly
 - **typescript**: [version from the `package.json`] 5.3.3
 - **eslint**: [version from the `package.json`] 8.56.0
 - **webpack**: [version from the `package.json`] "@quasar/app-webpack": "^3.12.3",
 - **os**: [e.g. Ubuntu 19.04] 23.10

**Comment by toresenneseth** [comment]
Same issue here. Anyone found a fix / workaround?

**Comment by ThulinaWickramasinghe** [comment]
I'm facing a similar issue but I'm using "react": "^18.1.0", with storybook and webpack.

![image](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/assets/68442470/d9573ae0-aecb-41ab-82c1-ac99d0f2ccdf)

**Comment by dominik-zeglen** [comment]
Hey, I've got the same problem in `6.4.2` but it stopped happening after updating to `6.5.3`

**Comment by brayovsky** [comment]
The issue and other typescript 5 related issues were fixed in version `6.5.3`.

**Comment by coderwei99** [comment]
i demotion typescript from 5.x to 4.x to resolve this.

**Comment by piotr-oles** [comment]
As @brayovsky said this was fixed in `6.5.3`, the quasar framework uses `6.5.0`

<img width="582" alt="image" src="https://github.com/user-attachments/assets/b4dfa72c-248f-4a4c-b044-4d488f747489">

I have no control over their dependencies, please create an issue on their repo :)
